rmgpy/kinetics/falloff.pyx

```diff
@@ -389,15 +401,30 @@ cdef class Troe(PDepKineticsModel):
         """
         import cantera as ct
 
-        assert isinstance(ct_reaction, ct.FalloffReaction), "Must be a Cantera FalloffReaction object"
+        assert isinstance(ct_reaction.rate, ct.TroeRate), "Must have a Cantera TroeRate attribute"
+
         ct_reaction.efficiencies = PDepKineticsModel.get_cantera_efficiencies(self, species_list)
-        ct_reaction.high_rate = self.arrheniusHigh.to_cantera_kinetics()
-        ct_reaction.low_rate = self.arrheniusLow.to_cantera_kinetics()
+
+        high_rate = self.arrheniusHigh.to_cantera_kinetics(ArrObj=True)
+        low_rate = self.arrheniusLow.to_cantera_kinetics(ArrObj=True)
         A = self.alpha
         T3 = self.T3.value_si
         T1 = self.T1.value_si
         if self.T2 is None:
-            ct_reaction.falloff = ct.TroeFalloff(params=[A, T3, T1])
+            falloff = [A, T3, T1]
         else:
             T2 = self.T2.value_si
-            ct_reaction.falloff = ct.TroeFalloff(params=[A, T3, T1, T2])
+            falloff = [A, T3, T1, T2]
+
+        ct_reaction.rate = self.to_cantera_kinetics(low_rate,high_rate,falloff) 
+
+    def to_cantera_kinetics(self, low, high, falloff): 
```

Comment:
This method `to_cantera_kinetics` dosn't refer to `self` and can't actually turn itself into cantera kinetics. i.e. a fully set up `Troe` object `my_troe` cannot do `my_troe.to_cantera_kinetics()` (unlike similar methods in other classes).
I think it should either be deleted and the one line it does just put in the only place it is called (line 419), or else it should be made to function just from `to_cantera_kinetics(self)`.